packages/calling-component-bindings/src/errorBarSelector.ts

```diff
@@ -114,10 +114,24 @@ export const errorBarSelector: ErrorBarSelector = createSelector(
       }
     }
 
```

Comment:
Can't comment on the line, too far up, but L86 I think should also have the macOS check as we are providing a mac speficifc string:
```
    if (diagnostics?.media.latest.microphonePermissionDenied?.value === true) {
      activeErrorMessages.push({ type: 'callMacOsMicrophoneAccessDenied' });
    }
```